docs/conf.py

```diff
@@ -318,8 +318,8 @@
     "postman_response": "![](../_static/img/postman_response.png)",
     "postman_retain_headers": "![](../_static/img/postman_retain_headers.png)",
     "fawrench": '<span class="fa fa-wrench" style="font-size: 1.6em;"></span>',
-    "SUPPORTED_PYTHON_VERSIONS_PLONE60": "3.8, 3.9, 3.10, 3.11, or 3.12",
-    "SUPPORTED_PYTHON_VERSIONS_PLONE61": "3.10, 3.11, or 3.12",
+    "SUPPORTED_PYTHON_VERSIONS_PLONE60": "3.9, 3.10, 3.11, 3.12, or 3.13",
```

Comment:
When we drop a Python version, I think of all the places where we should add an admonition. For the drop of Python 3.8 in 6.0.14, where would be good places?

I'll push a couple of commits for discussion. If they're not appropriate, we can change or drop them.

Reply:
It looks like the include was the place for this, and it gets pulled into three places.

Reply:
LGTM, thanks!